src/sentry/snuba/events.py

```diff
@@ -675,21 +703,23 @@ class Columns(Enum):
         event_name="contexts[trace.trace_id]",
         transaction_name="trace_id",
         discover_name="contexts[trace.trace_id]",
-        issue_platform_name="contexts[trace.trace_id]",
+        issue_platform_name="trace_id",
```

Comment:
This was already lifted to a separate column in the dataset so we shouldn't have been indexing into `context`.